### ISSUE TITLE
PHI-Compatible Cloud Computing Environment

### ISSUE THREAD

**Issue by jonc101** [HealthRex/CDSS#75]
Stanford doesn't have BAA with AWS yet, and could be 2019 before is usable.
Have with Google Cloud Platform now, but then have to go through IRT central account agreement.

Have Nero access now, though not much experience in using it.

Confusing mix of
- Google Cloud + BigQuery general use that anyone can login to
- GCP that requires SUNetID / @stanford.edu account to access
- GCP that requires Stanford VPN (Cisco anywhere) login first before can access VPC?
- Nero GCP/BQ that is safe for PHI that apparently is not any of the above???

Notes from conversation with Tina Seto

jonathan_h_chen 9:57 AM
Is there a starting point for documentation or point of contact for Nero BQ? I'm getting confused on what I'm on now, what's requiring VPN, and what still counts separately as Nero BQ?
9:57 AM
I had an analyst look into some of this a while ago, but he's out now.

Tina Seto 9:58 AM
Alternatively it might be okay if you can get Garrick to whitelist your existing BQ into VPC controls like in the doc.

9:58 AM
Please work with Garrick or Somalee directly on that if it seems an easier path than Nero

jonathan_h_chen 10:11 AM
Which Garrick is it?

Tina Seto 10:12 AM
Garrick Olson, our RIT architect. Or try Priya Desai (prd@stanford.edu). She wrote the doc and is a bit more responsive

jonathan_h_chen 10:14 AM
Great, thanks. I think you said you were going to email a doc as well? Don't think I saw it yet

new messages

Tina Seto 10:14 AM
The doc link re. VPC controls.. that's the only one.
10:15 AM
https://docs.google.com/document/d/10lNrDJR5QUZE643mpLy5OTGN4WjqlJnVUlNYgyellwU/edit

Notes from Neil Soderquist regarding Nero:
Hi Jonathan, 

You made a fairly accurate bullet list - generally I can only speak on the requirements of Nero as what the SoM does with their services may be different. We suggest connecting to the VPN if you plan on working with Nero or Nero related services (buckets, etc). Even though you may be on campus already, the VPN is still required to access Nero services - it's just another security precaution. I do understand how it can be burdensome at times, but it's a requirement to work with the high-risk data stored in the Nero environment(s). 

With Nero GCP, projects have the VPC Access Control, VPN, Two-Step Auth, and project/resource permissions themselves that act as layers of security. All Nero projects are put behind the VPC, which requires full-traffic VPN. Resources in that project also require VPN and prompt for password and two-step authentication.

Simply put, if you're on the full-traffic VPN you should be able to access any Nero resource (on-prem or GCP).

Levels of security applied typically depend on the data risk classification involved with the service to adhere with minimum security requirements set by the Information Security Office. (https://uit.stanford.edu/guide/riskclassifications, https://uit.stanford.edu/guide/securitystandards#security-standards-servers)

Best,
Neal
